app/2.0/docs/tools/polymer-cli.md

```diff
@@ -39,500 +39,28 @@ You're all set. Run `polymer help` to view a list of commands.
 
 ## Overview {#overview}
```

Comment:
I know this organization is not new, but it strikes me as strange that the overview follows the install. 

Maybe move the first sentence to the head of the article (before the Install heading) and retitle this section "Types of projects" or "CLI project types".